docs/user-guide/ze-profiles.md

```diff
@@ -2,10 +2,119 @@
 
 After you install Zowe Explorer, you need to have a Zowe Explorer profile to use all functions of the extension. You can optionally activate the Secure Credential Store plug-in to securely store your credentials.
 
-**Note:** You can continue using Zowe V1 profiles with the Zowe Explorer V2. For more information, see ***insert link here***.
+**Note:** You can continue using Zowe V1 profiles with Zowe Explorer V2. 
+## Configuring team profiles
+
+Zowe CLI team profiles simplify profile management by letting you to edit, store, and share mainframe configuration details in one location. You can use a text editor or an IDE to populate configuration files with connection details for your mainframe services. By default, your team configuration file is located in the `.zowe home` folder, whereas the project-level configuration file is located in the main directory of your project.
+
+**Note**: A project context takes precedence over global configuration.
+### Creating team configuration files
+
+Create a team configuration file.
+
+1. Navigate to the explorer tree.
+2. Hover over **DATA SETS**, **USS**, or **JOBS**.
+3. Click the **+** icon.
+4. Select **Create a New Team Configuration File**.
+5. Chose either a global configuration file or a project-level configuration file.
+6. Edit the config file to include the host information.
+7. Refresh Zowe Explorer by either clicking the button in the notification message shown after creation, `alt+z`, or the `Zowe Explorer: Refresh Zowe Explorer` command palette option.
+
+Your team configuration file appears either in your `.zowe` folder if you choose the global configuration file option, or in your workspace directory if you choose the project-level configuration file option. The notification message that shows in VS Code after config file creation will include the path of the file created.
+
+### Managing profiles
+
+You can edit your project-level or global configuration files.
+
+**Follow these steps**:
+
+1. Right-click on your profile.
+2. Select the **Add**, **Update**, or **Delete Profile** options to edit the zowe config file in place.
+
+   **Tip:** Use the Intellisense prompts if you need assistance with filling parameters in the .json file.
```

Comment:
We should probably avoid saying `.json` file.
Maybe we just say "the file" for now ?
I know this is something we have to review across the Client components for Zowe V2 (after the GA date)

Reply:
removed  .json